### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by matthewbloch**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,45 +1,49 @@
...
+LINKFLAGS      := -lz -lSDL2 -lm -ldl
+LINKFLAGS_WASM := -sUSE_SDL=2 -sALLOW_MEMORY_GROWTH=1 -sFORCE_FILESYSTEM -sEXPORTED_RUNTIME_METHODS=[\"ccall\"] -lidbfs.js
+DATA           := ddnoise roms/riscos311/ros311 roms/arcrom_ext cmos arc.cfg
 ifdef DEBUG
   CFLAGS += -D_DEBUG -DDEBUG_LOG -g3
 #  --source-map-base not needed if .map is in the same dir as .wasm
   LINKFLAGS_WASM += -gsource-map
   BUILD_TAG +=  (DEBUG)
   $(info ❗BUILD_TAG="${BUILD_TAG}")
-  DATA += roms/riscos311/ros311 roms/arcrom_ext cmos arc.cfg
```
Comment (pdjstone) [inline Makefile:18]: When doing an emscripten build as part of archimedes live we don't want to include ROMs in the build package, as they're loaded dynamically. I know you had a minimal emscripten build working with the default emscripten UI. Maybe that could be a separate make target?

Comment (matthewbloch) [reply Makefile:18]: Ah OK that must have broken somehow, and I couldn't work out why that wasn't the default :) Feel free to back it out, I was thinking about adding a bit more internal abstraction around loading support & config files anyway.

**Comment by pdjstone** [comment]
Thanks for this. Both `make wasm` and `make native` still work fine. Both the native version or the minimal Emscripten version still boot, however it looks like I have at some point broken the mouse/keyboard input on these  There are now some assumptions in the C code that it is being run inside the Archimedes Live UI.

**Comment by matthewbloch** [comment]
Ah good wasn't just me then 😅

**Pull request merged**